src/angular-vs-repeat.js

```diff
@@ -144,7 +144,7 @@
         return {
             restrict: 'A',
             scope: true,
-            compile: function($element, $attrs) {
+            compile: function($element, $attrs, transclude) {
```

Comment:
According to the angular docs, we shouldn't use the transclude function passed to compile, we should use the one passed to link instead. 

From: https://docs.angularjs.org/api/ng/service/$compile
> Note: The transclude function that is passed to the compile function is deprecated, as it e.g. does not know about the right outer scope. Please use the transclude function that is passed to the link function instead.

Instead this should go down on line 207, replacing 
``` js
pre: function($scope, $element, $attrs) {
```
with this:
``` js
pre: function($scope, $element, $attrs, _controller, transclude) {
```